libbeat/publisher/queue/memqueue/consume.go

```diff
@@ -48,6 +46,7 @@ func newConsumer(b *broker) *consumer {
 	}
 }
 
+/*
```

Comment:
Can this commented out section just be removed?

Reply:
Yes, done :-)